docs/en/advanced_guides/detection_transformer.md

```diff
@@ -0,0 +1,230 @@
+# Detection Transformer
+
+Detection Transformers (DETRs) are a series of Transformer-based object detectors.
+
+In most DETRs, the features extracted from backbone and neck were fed into a Transformer which is composed of an encoder and a decoder. The Transformer directly outputs a set of queries in parallel. Each query corresponds to one prediction, which may be an object or `no object`.
+
+![DETR_overall](.\DETR_overall.png)
```

Comment:
use a githubusercontent link instead of a png file?

Reply:
> use a githubusercontent link instead of a png file?

OK~ Thanks very much for your review~